grouper/graph.py

```diff
@@ -130,29 +130,12 @@ def _get_checkpoint(session):
         return counter.count, int(counter.last_modified.strftime("%s"))
 
     @staticmethod
-    def _get_disabled_users(session):
-        '''
-        Returns a dict of username: [ list of public keys ].
-        '''
-        out = {}
-        users = session.query(User).filter(
-            User.enabled == False
-        )
-        for user in users:
-            out[user.username] = [{
-                "public_key": key.public_key,
-                "fingerprint": key.fingerprint,
-                "created_on": str(key.created_on),
-            } for key in user.my_public_keys()]
-        return out
-
-    @staticmethod
-    def _get_user_metadata(session):
+    def _get_user_metadata(session, enabled=True):
```

Comment:
Why don't we get all metadata? The query is more efficient to run once, and since there's no index on the column, it's a full table scan either way.